ui/e2e/src/fixtures/dashboardTest.ts

```diff
@@ -72,7 +72,7 @@ async function duplicateDashboard(projectName: string, dashboardName: string, ne
   const result = await createDashboard(newDashboardJson);
   if (!result.ok) {
     throw new Error(
-      `Unable to create test dashboard '${newDashboardName}'. Failed with status '${result.status}: ${result.statusText}'.`
+      `Unable to create test dashboard '${newDashboardName}'. Failed with status '${result.status}: ${result.statusText}'. \n${await result.text()}`
```

Comment:
Really useful?

Reply:
Yes for the tests, when we see a 400 we can know why we see a 400 this is a game changer!